FreeSurfer/custom/SetUpFSNHP.sh

```diff
@@ -0,0 +1,270 @@
+  VariableSigma="3"
+  PialSigma="2"
+  SmoothNiter="1"
+  if ((isFLAIR)) ; then                                  # control T2 pial
+    NSigmaAbove="3"                                      # 2: FS6 default
+    NSigmaBelow="3"                                      # 3: FS6 default
+  else
+    NSigmaAbove="3"                                      # 3: FS6 default
+    NSigmaBelow="3"                                      # 3: FS6 default
+  fi
+  WMProjAbs="1"    
+  MaxThickness="8"                                            # FS6 conf2hires       
+  CopyBiasFromConf="TRUE"                                     # FS6 conf2hires
+
+  VariableSigmaFS5="8"                                        # FS5.3.0 HighResPial  
+  MaxThicknessFS5="5"                                         # FS5.3.0 HighResPial    
+  GreySigmaFS5="5"                                            # FS5.3.0 HighResPial
+  GCAdir="${HCPPIPEDIR_Templates}/ChimpYerkes29"
+  GCA="RB_all_2008-03-26.gca"
+  AvgCurvTif="average.curvature.filled.buckner40.tif"
+
+  # FS BBR in DiffusionPreprocPipeline
+  FSBBRDIFF=TRUE
+  DiffWMProjAbs="1"
+
+elif [[ "$SPECIES" == *Macaque* ]] ; then                            # tuned by TH and AU
+
+  # IntensityCor
+  IntensityCor="FAST"
+  ScaleFactor="2"
+  BiasFieldFastSmoothingSigma="40"                            # = 20*ScaleFactor
+  StrongBias_args="-s"                                        # strong bias in T1w
+
+  # recon-all.v6.hiresNHP 
+  mri_normalize_args="-sigma 16 -gentle"                      # = 8*ScaleFactor. FS6 default sigma=8
+
+  mris_make_surfaces_args=""
+  initcc="128 98 124"                                         # FS5.3.0 voxel coordinate in orig.mgz and wm.mgz
+  mri_fill_args="-C 0 -3 8 -fillven 1 -topofix norm.mgz"      # mm coordinate in orig.mgz and wm.mgz
+  mris_inflate1_args="-n 250"                                 # set higher # of iterations (default=10) to avoid "white surface short cut"
+  mris_sphere_args=" -RADIUS 55 -remove_negative 1"
+
+  if [[ "$SPECIES" == *30BS* ]] ; then                          # Rhesus and Cyno hybrid template
+
+    BrainTemplate="Mac30BS"
+    GCAdir="${HCPPIPEDIR_Templates}/NHP_NNP/${BrainTemplate}/fsaverage" #Template Dir with FreeSurfer NHP GCA and TIF files
+    TemplateWMSkeleton="${HCPPIPEDIR_Templates}/NHP_NNP/${BrainTemplate}/MNINonLinear/${BrainTemplate}_wmskeleton.nii.gz"
+
+  elif [[ "$SPECIES" == *Cyno* ]] ; then                        # Cynomolgus (Macaca fascicularis)
+
+    #WMSeg_wlo="95"                                      # suppress white matter surface invasion to cortical ribbon  e.g. M174
+    #mri_segment_args="-wlo $WMSeg_wlo" 
+    BrainTemplate="Mac25Cyno"
+    GCAdir="${HCPPIPEDIR_Templates}/NHP_NNP/${BrainTemplate}/fsaverage"
+    TemplateWMSkeleton="${HCPPIPEDIR_Templates}/NHP_NNP/${BrainTemplate}/MNINonLinear/${BrainTemplate}_wmskeleton_0.5mm.nii.gz"
+
+  elif [[ "$SPECIES" == *Rhesus* ]] ; then                      # Rhesus (Macaca mulatta)
```

Comment:
The NHP pipeline help says that `MacaqueRhes` is a valid setting, but it will not test true here.